janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -497,10 +499,17 @@ private void mutateManyLogged(final Map<String, Map<StaticBuffer, KCVMutation>>
                 final StaticBuffer key = keyAndMutations.getKey();
                 final KCVMutation keyMutations = keyAndMutations.getValue();
 
-                final Iterator<BatchableStatement<BoundStatement>> deletions = Iterator.of(commitTime.getDeletionTime(this.times))
+                Iterator<BatchableStatement<BoundStatement>> deletions;
+                Iterator<BatchableStatement<BoundStatement>> additions;
+                if (commitTime != null) {
```

Comment:
That sounds like a fair statement, but on the other hand, it's usually easier to understand "something exists" than "something does not exist". I agree that `a==b` is easier to understand then `a!=b`, but when it comes to `a==null` vs `a!=null`, it may not be that evident, at least for me.

Reply:
Makes sense to me. Thank you! Resolving this comment